hello.js

```diff
@@ -0,0 +1,3 @@
+function() {
```

Comment:
hey @vmarkovtsev, why doesn't this code raise any concerns?

the formatting is clearly wrong

Reply:
This does not work this way; we never fix **all** the formatting faults, at least this year.
Besides, the repo is quite small and hence there are few rules mined.

At this development stage, we should fix **something** **sometimes**, but be super precise with that.

Reply:
I trained the model for this repo and master branch and get 3 rules. All these rules make sense, but their confidence is `0.915`, `0.904` and `0.904`. That means that rules are about 90% sure that they are correct. However, we have confidence threshold set on `0.92`. That means that these rules are disabled. 

And before you ask, I can tell that we cannot make a threshold lower because it leads to comment quality drop: we begin to see mistakes in correct style. 

---
Rules
```
Confidence: 0.915. Support: 841
     -1.reserved = {
   ∧ -1.roles not in {STRING}
   ∧ +1.internal_type not in {StringLiteral}

Confidence: 0.904. Support: 757
     -1.reserved = {
   ∧ -1.roles not in {STRING}
   ∧ +1.roles not in {STRING}

Confidence: 0.904. Support: 766.
     -1.internal_type not in {StringLiteral}
   ∧ -1.reserved = {
   ∧ +1.roles not in {STRING}
```